std-bits/table/src/main/java/org/enso/table/data/column/builder/InferredBuilder.java

```diff
@@ -96,7 +96,7 @@ public void appendBulkStorage(Storage<?> storage) {
     }
   }
 
-  private void initBuilderFor(Object o) {
+  public void initBuilderFor(Object o) {
```

Comment:
I'm not sure exposing this internal implementation detail is the right way to do this.

I guess it's acceptable, but I think we can do it simpler.

Reply:
Could we not move this to `Builder.java` and have a public method there `getBuilderFor(Object o, int capacity)` and use that here?

Reply:
We already have `Builder::getForType` there, which I suggested that we should use.

I guess we could refactor `InferredBuilder::initBuilderFor` to _also_ rely on `Builder::getForType`, to keep this logic in a single place. But `initBuilderFor` does some additional logic too (adding a prefix of NULL values, figuring out the initial size), so that would have to be kept separate anyway.

Reply:
Done -- sorry, I missed the discussion where you mentioned getForType, I just took a guess about what to use.